### ISSUE TITLE
Coinex ImplicitAPI class misses  cancel stop loss and take profit api end points

### ISSUE THREAD

**Issue by BertkeOfUtrecht** [ccxt/ccxt#27832]
### Operating System

Windows

### Programming Language

Python

### CCXT Version

4.5.35

### Description

Coinex ImplicitAPI class should have below two end points supported which are missing currently.

After support being implemented, below two functions will be available in coinex.py ImplicitAPI class

    v2_private_post_futures_cancel_position_stop_loss = v2PrivatePostFuturesCancelPositionStopLoss = Entry('futures/cancel-position-stop-loss', ['v2', 'private'], 'POST', {'cost': 20})
    v2_private_post_futures_cancel_position_take_profit = v2PrivatePostFuturesCancelPositionTakeProfit = Entry('futures/cancel-position-take-profit', ['v2', 'private'], 'POST', {'cost': 20})

**Comment by carlosmiei** [comment]
Hello @BertkeOfUtrecht thanks for reporting it, we will be adding them soon

**Comment by carlosmiei** [comment]
@BertkeOfUtrecht We have released a new version with the fix!
